Solutions/ZeroNetworks/SolutionMetadata.json

```diff
@@ -0,0 +1,21 @@
+{
+	"publisherId": "azuresentinel",
+	"offerId": "azure-sentinel-solution-znaccessochestratoraudit",
+	"firstPublishDate": "2022-06-06",
+	"lastPublishDate": "2022-06-06",
+	"providers": [
+		"Zero Networks"
+	],
+	"categories": {
+		"domains": [
+			"Security - Network"
+		],
+		"verticals": []
+	},
+	"support": {
+		"name": "Zero Networks",
+		"email": "support@zeronetworks.zendesk.com",
+		"tier": "community",
+		"link": "https://github.com/Azure/Azure-Sentinel"
```

Comment:
@dicolanl, link should be the support link, thanks.

Reply:
fixed